conversation.go

```diff
@@ -0,0 +1,14 @@
+package hubspot
+
+type Conversation struct {
+	VisitorIdentification      VisitorIdentificationService
+	IdentificationTokenRequest IdentificationTokenRequest
```

Comment:
`IdentificationTokenRequest` needs to be switched on a request-by-request basis, so I don't think it's necessary as a field.